org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/clientnotification/ClientNotificationPoller.java

```diff
@@ -70,7 +70,7 @@ public void stop() {
       return;
     }
 
-    LOG.debug("Stopping client notification poller [clientNodeId={}].", INode.ID);
+    LOG.debug("Stopping client notification poller [clientNodeId={}].", NodeId.current());
```

Comment:
Use IIds.toString(NodeId.current()) to avoid logging "NodeId" class name, see org.eclipse.scout.rt.dataobject.id.AbstractId#toString